pkg/kv/kvprober/kvprober.go

```diff
@@ -419,7 +449,14 @@ func (p *Prober) readProbeImpl(ctx context.Context, ops proberOpsI, txns proberT
 	}
 
 	d := timeutil.Since(start)
-	log.Health.Infof(ctx, "kv.Get(%s), r=%v returned success in %v", step.Key, step.RangeID, d)
+	// Extract leaseholder information from the trace recording if enabled
```

Comment:
We will want this logic for the write probing also.

Reply:
Yes, I have added it for write probes too now. Thank you!